src/types.ts

```diff
@@ -582,7 +582,7 @@ export class Internal {
     for (const key in fixedArg) {
       if (!name.includes('guild') && key.endsWith('_id')) {
         const value = +fixedArg[key]
-        if (BigInt(Math.abs(value)) < 1n << 32n) {
+        if (Number.isInteger(value) && BigInt(value) < 1n << 31n && BigInt(value) >= -(1n << 31n)) {
```

Comment:
似乎从一开始就可以直接去掉 BigInt，写成

```suggestion
        if (Math.abs(value) < 4294967296) {
```